upm-tools/AssemblyDefinitions/Bugsnag.asmdef

```diff
@@ -0,0 +1,3 @@
+﻿{
```

Comment:
Should the AssemblyDefinitions be checked-in or are they generated?

Reply:
they should be checked in